Makefile

```diff
@@ -0,0 +1,73 @@
+LICENSE_CONFIG?="license-config.json"
+
+LICENSE:
+	@echo "you must have a LICENSE file" 1>&2
+	exit 1
+
+LICENSE_HEADER:
+	@echo "you must have a LICENSE_HEADER file" 1>&2
+	exit 1
+	
+	
+.PHONY: pretty
+pretty: node_modules/prettier
+	npm run format
+
+.PHONY: lint-fix
+lint-fix: node_modules/eslint
+	npm run lint:fix
+
+node_modules/eslint:
+	npm ci
+
+node_modules/mocha:
+	npm ci
+
+node_modules/prettier:
+	npm ci
+
+node_modules/typescript:
+	npm ci
+
+node_modules/license-check-and-add:
+	npm ci
+
+run-env-%:
+	export ENV=$* \
+	&& export NODE_PATH=$(DIR) \
+	&& cp $(PROJECT_ROOT)/env/$(ENV)/.env .env \
+	&& npm run start
```

Comment:
run start probably doesnt make sense for lambdas?

Reply:
will remove